src/components/docs-page/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactElement } from 'react'
+import { MDXRemote } from 'next-mdx-remote'
+import defaultMdxComponents from '@hashicorp/platform-docs-mdx'
+import Placement from 'components/author-primitives/shared/placement-table'
+import NestedNode from 'components/author-primitives/waypoint/nested-node'
+
+const additionalComponents = defaultMdxComponents('waypoint', {
+  Placement,
+  NestedNode,
+})
+
+const DocsPage = (props): ReactElement => {
+  return <MDXRemote {...props} components={additionalComponents} />
```

Comment:
💭 We may or may not want the ability to pass in custom components per product. Should be easy enough to add if we do want to continue supporting that, but I don't think we need to adjust anything righ tnow.